Makefile

```diff
@@ -1,24 +1,49 @@
-REQUIREMENTS=requirements.txt
-REQUIREMENTS_CEXT=requirements/cext.txt
-REQUIREMENTS_CEXT_NOARCH=requirements/cext_noarch.txt
-
 .PHONY: help clean clean-pyc clean-build list test test-all coverage docs release sdist
 
 help:
-	@echo "clean-build - remove build artifacts"
-	@echo "clean-pyc - remove Python file artifacts"
-	@echo "lint - check style with flake8"
-	@echo "test - run tests quickly with the default Python"
-	@echo "testall - run tests on every Python version with tox"
-	@echo "coverage - check code coverage quickly with the default Python"
-	@echo "docs - generate Sphinx HTML documentation, including API docs"
-	@echo "release - package and upload a release"
-	@echo "dist - package"
-	@echo "writeversion - updates the kolibri/VERSION file"
-
-clean: clean-build clean-pyc clean-docs clean-static
-
-clean-static:
+	@echo "Usage:"
+	@echo ""
+	@echo "make <command>"
+	@echo ""
+	@echo "Building"
+	@echo "--------"
+	@echo ""
+	@echo "dist: create distributed source packages in dist/"
+	@echo "pex: builds a portable .pex file for each .whl in dist/"
+	@echo "assets: builds javascript assets"
+	@echo "staticdeps: downloads/updates all static deps bundled into the dist"
+	@echo "staticdeps-cext: downloads/updates C extensions for all supported platforms"
+	@echo "clean: restores code tree to a clean state"
+	@echo "clean-build: remove build artifacts"
+	@echo "clean-pyc: remove Python file artifacts"
+	@echo "clean-assets: removes javascript assets"
+	@echo "writeversion: updates the kolibri/VERSION file"
+	@echo "release: package and upload a release"
+	@echo "buildconfig: [unsupported] runs a special script for building a source package with special requirements.txt"
+	@echo ""
+	@echo "Development"
+	@echo "-----------"
+	@echo ""
+	@echo "lint: check style with flake8"
+	@echo "test: run tests quickly with the default Python"
+	@echo "test-all: run tests on every Python version with tox"
+	@echo "coverage: check code coverage quickly with the default Python"
```

Comment:
for test-related commands, let's specify when they also run JS tests

Reply:
Sure thing, I don't see any of the JavaScript testing stuff recorded in the Makefile.. guess frontenders prefer other ways? :)